checkov/common/checks_infra/checks_parser.py

```diff
@@ -252,6 +253,16 @@ def _parse_raw_check(self, raw_check: Dict[str, Any], resources_types: Optional[
         else:
             resource_type = raw_check.get("resource_types", [])
             if (
+                    resource_type is not None and
```

Comment:
Any reason not to replace `resource_type is not None and resource_type != []` with `resource_type` ?

Reply:
Nope. I will adjust.